### PR TITLE
Update scholarX 2020 mentors details

### LINKED ISSUE #614: Update scholarX 2020 mentors details
**Is your feature request related to a problem? Please describe.**
- Few more changes needed to be added to the mentors.

**Describe the solution you'd like**
- Update the `mentors.json` file using given details.
https://docs.google.com/spreadsheets/d/1zOEfTBTTBDdDKcafHI44CY73v2obLEnnOupNBzzO0Do/edit#gid=193573752
- Highlighted everything needs to be changed/added in blue. (Most important is to make the spelling correction highlighted in red text!)

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #614 

## Goals

- Update scholarX 2020 mentors details

## Approach

- Update the `mentors.json` file using given details.
  
### Preview Link

https://pr-615-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by Gravewalker666 — APPROVED** [review]
Great job @anjula-sack

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Nishantha Rajakaruna's website button is not rendered properly.

Screenshot 2020-03-20 at 16 39 28

cc: @Gravewalker666 @Piumal1999

File: src/main/webapp/scholarx/2020/mentors.json [hunk 1]
```diff
@@ -331,7 +331,18 @@
...
-    "socialLinks" : []
+    "socialLinks" : [
+      {
+        "platform" : "facebook",
+        "icon" : "fa-facebook",
+        "url" : "https://www.facebook.com/nishanta.rajakaruna.7"
+      },
+      {
+        "platform" : "website",
+        "icon" : "fa-globe",
```
Comment (Gravewalker666) [inline src/main/webapp/scholarx/2020/mentors.json]: - `<i class="fas fa-globe"></i>`
This is the full HTML line for the globe icon. I think the issue is that `fas` class at the beginning is not the one in the template.

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

**Review by Gravewalker666 — DISMISSED** [review]

**Comment by jayasanka-sack** [comment]
The button is still invisible. Only the icon is there.

Please go through the template first.
1. 
 https://github.com/sef-global/sef-site/blob/4f5320033dba376a762114727745bccb2175d469/src/main/webapp/scholarx/2020/index.html#L628

the key `platform` is to render the necessary CSS class for the button.
The value `website` becomes `btn-website` but, there's no such a class defined. Just go through available btn classes and add a one. 

2. 
https://github.com/sef-global/sef-site/blob/4f5320033dba376a762114727745bccb2175d469/src/main/webapp/scholarx/2020/index.html#L629

The icon loads this way.  `fas fa-globe` becomes `<i class="fab fas fa-globe"></i>` which isn't nice.

Please edit the template as `<i class="{{icon}}"></i>` and modify all the icon keys' values with the prefix.

ex: 
```
{
...
"icon" : "fab fa-facebook",
...
}
```

**Review by jayasanka-sack — APPROVED** [review]
LGTM

**Review by Gravewalker666 — APPROVED** [review]

**Pull request merged**